tools/projmgr/schemas/common.schema.json

```diff
@@ -127,8 +127,8 @@
     },
     "CompilerType": {
       "type": "string",
-      "pattern": "^(GCC|CLANG|AC6|IAR)(@(>=)?([0-9]+\\.[0-9]+\\.[0-9]+((\\+|\\-)[a-zA-Z0-9_\\.\\+-]+)?))?$",
-      "description": "Selection of the toolchain used in the project (GCC, CLANG, AC6, IAR), optionally with version, for example AC6@6.16.0"
+      "pattern": "^(GCC|CLANG|AC6|IAR|TIARMCLANG)(@(>=)?([0-9]+\\.[0-9]+\\.[0-9]+((\\+|\\-)[a-zA-Z0-9_\\.\\+-]+)?))?$",
+      "description": "Selection of the toolchain used in the project (GCC, CLANG, AC6, IAR, TIARMCLANG), optionally with version, for example AC6@6.16.0"
```

Comment:
It seems in CMake this compiler is identified as `TIClang`:
https://cmake.org/cmake/help/latest/variable/CMAKE_LANG_COMPILER_ID.html
Instead of `TIARMCLANG` maybe we could make it shorter `TICLANG`?

Reply:
Ok.